rest/data/project_test.go

```diff
@@ -283,3 +284,92 @@ func (s *ProjectConnectorGetSuite) TestFetchKeyOutOfBoundDesc() {
 	s.NoError(err)
 	s.Len(projects, 0)
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// Tests project create action
+type ProjectConnectorCreateUpdateSuite struct {
+	ctx      Connector
+	setup    func() error
+	teardown func() error
+	suite.Suite
+}
+
+func TestProjectConnectorCreateUpdateSuite(t *testing.T) {
+	s := new(ProjectConnectorCreateUpdateSuite)
+	s.setup = func() error {
+		s.ctx = &DBConnector{}
+		testutil.ConfigureIntegrationTest(t, testConfig, "TestProjectConnectorCreateUpdateSuite")
+		db.SetGlobalSessionProvider(testConfig.SessionFactory())
+		return nil
+	}
+
+	s.teardown = func() error {
+		return db.Clear(model.ProjectRefCollection)
+	}
+
+	suite.Run(t, s)
```

Comment:
Unless you want to run the same test with different fixtures, I'd just put these in the setup/teardown methods.

Reply:
@tychoish , can you please give me a hint? I am total noob in go tests,

Reply:
If you create a function named SetupSuite and TearDownSuite (case sensitive), they will run before/after each test. Look for those in existing code for an example

Reply:
Sorry for not following up on this. John is correct.